src/component/mxgraph/config/ShapeConfigurator.ts

```diff
@@ -36,47 +38,53 @@ import { ComplexGatewayShape, EventBasedGatewayShape, ExclusiveGatewayShape, Inc
 import { TextAnnotationShape } from '../shape/text-annotation-shapes';
 import { BpmnStyleIdentifier } from '../style';
 
+const registerShapes = (): void => {
```

Comment:
**thought:** we could register the shapes only once if necessary. The shapes are registered in a global state in mxGraph, so bpmn-visualization is registered the same shapes at every instantiation of the `BpmnVisualization` class.
This is what is done in maxGraph: https://github.com/maxGraph/maxGraph/blob/v0.8.0/packages/core/src/view/cell/register-shapes.ts#L36-L43